{{cookiecutter.project_slug}}/.dockerignore

```diff
@@ -2,3 +2,4 @@
 !.coveragerc
 !.env
 !.pylintrc
+node_modules/
```

Comment:
This must be made conditional depending on presense or absense of `node_modules/`, for instance, this is the case when `cookiecutter.js_task_runner == "y"`. Other cases need to be thought over as well.